Ghidra/Framework/SoftwareModeling/src/main/java/ghidra/pcodeCPort/slghpatexpress/OperandValue.java

```diff
@@ -50,6 +50,8 @@ public void changeIndex(int newind) {
 
 	@Override
 	public TokenPattern genMinPattern(VectorSTL<TokenPattern> ops) {
+		if (index > ops.size())
```

Comment:
Are these indices 0- or 1-based? I think `>` should be `>=` (but don't shoot me if I'm wrong).

Reply:
Yeah that clicked a bit after pushing, it's zero based. Was gonna fix when I got home, but thanks for sanity check